test/functional/gridfs.test.js

```diff
@@ -1,12 +1,12 @@
 'use strict';
 
-const test = require('./shared').assert,
-  setupDatabase = require('./shared').setupDatabase,
-  fs = require('fs'),
-  format = require('util').format,
-  child_process = require('child_process'),
-  expect = require('chai').expect,
-  Buffer = require('safe-buffer').Buffer;
+const { setupDatabase, assert: test } = require('./shared');
+const fs = require('fs');
+const { format } = require('util');
+const child_process = require('child_process');
+const { expect } = require('chai');
+const { Buffer } = require('safe-buffer');
```

Comment:
We still run tests in node v4 environments for 3.6 so we can't use destructuring here.